openpype/hosts/resolve/api/lib.py

```diff
@@ -266,28 +272,41 @@ def create_timeline_item(media_pool_item: object,
     clip_name = _clip_property("File Name")
     timeline = timeline or get_current_timeline()
 
+    # timing variables
+    fps = project.GetSetting("timelineFrameRate")
```

Comment:
A resole timeline can deviate from the project FPS.
E.g. if Create New Timeline had Use Project Settings enabled bottom left:

![image](https://github.com/ynput/OpenPype/assets/2439881/5331ffbf-0704-4080-bfb2-66303f7a0e0e)

This should instead be getting the setting from the timeline:
```suggestion
    fps = timeline.GetSetting("timelineFrameRate")
```

Reply:
fixed here a11467883b5d1cec302f43361e7d939ff502ac07